gomint-server/src/main/java/io/gomint/server/world/block/Border.java

```diff
@@ -0,0 +1,16 @@
+package io.gomint.server.world.block;
+
+import io.gomint.world.block.BlockType;
+
+public class Border extends Block implements io.gomint.world.block.BlockBorder {
+
+    @Override
+    public float getBlastResistance() {
+        return 0; //TODO get it , No Information
```

Comment:
This should have a very high (like bedrock) blast resistance

Reply:
yes :D